src/components/20-molecules/dropdown/story.js

```diff
@@ -20,7 +20,7 @@ storyDropdown.add('Dropdown', () => {
   const name = text('name', '');
   const invalid = boolean('invalid', false);
   const error = text('error', 'Error Message');
-  const native = boolean('native', true);
+  const native = boolean('native', false);
```

Comment:
Why this knobs default? I was quite surprised to find that my viewport changes did not toggle between native and enhanced mode, and only after a while discovered it was due to a knob default. This is bad UX in my opinion, so I suggest this line to be reverted.

Reply:
Done before you wrote that ;)

Reply:
native should be default false, isnt it?